site/gatsby-site/pages/apps/discover.js

```diff
@@ -0,0 +1,807 @@
+import React, { useState, useEffect, useRef, useMemo } from 'react';
+import { StringParam, QueryParams, useQueryParams } from 'use-query-params';
+import { MiddleLayout } from '@components';
+import algoliasearch from 'algoliasearch/lite';
+import { debounce } from 'debounce';
+import {
+  InstantSearch,
+  Highlight,
+  connectHits,
+  connectRefinementList,
+  Pagination,
+  Stats,
+  connectSearchBox,
+  connectCurrentRefinements,
+} from 'react-instantsearch-dom';
+import styled from 'styled-components';
+import config from '../../config';
+import md5 from 'md5';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faNewspaper,
+  faIdCard,
+  faUserShield,
+  faFlag,
+  faHashtag,
+  faTimesCircle,
+} from '@fortawesome/free-solid-svg-icons';
+import { useModal, CustomModal } from '../../src/components/useModal';
+
+import '../../static/discover/src/algolia.css';
```

Comment:
Are these necessary? If so, let's open an issue to remove their dependency. We don't want to depend on styling from old discover app since it will eventually be removed.